server/src/controllers/users.js

```diff
@@ -0,0 +1,35 @@
+import model from '../models';
+
+const { user } = model;
+
+class Users {
+  static async createUser(req, res) {
+    const {
+      firstName,
+      lastName,
+      email,
+      password,
+    } = req.body;
+
+    const newUser = await user.create({
+      firstName,
+      lastName,
+      email,
+      password,
+    });
+    res.status(201).json({
+      status: 201,
+      message: 'Account created successfuly',
+      data: newUser,
+    });
+  }
+
+  static async usersList(req, res) {
+    const allUsers = await user.findAll();
+    res.status(200).json({
+      status: 200,
+      data: allUsers,
```

Comment:
Is the user password part of the object that is being returned?

If it is, an alternative would be to define an object by yourself and return it instead of returning the response from the query directly.
E.g: data: {
             firstname,
             lastname,
             email
              }
I don't think it's safe to return password in the response

Reply:
Noted